build.py

```diff
@@ -0,0 +1,96 @@
+import glob
+import os
+import shutil
+import urllib.request
+import zipfile
+from distutils.dir_util import copy_tree
+
+def build_govuk_assets():
+    DIST_ROOT = "./app/static/dist"
+    GOVUK_DIR = "/govuk-frontend"
+    GOVUK_URL = (
+        "https://github.com/alphagov/govuk-frontend/"
+        "releases/download/v4.6.0/release-v4.6.0.zip"
+    )
+    ZIP_FILE = "./govuk_frontend.zip"
+    DIST_PATH = DIST_ROOT + GOVUK_DIR
+    ASSETS_DIR = "/assets"
+    ASSETS_PATH = DIST_PATH + ASSETS_DIR
+
+    # Checks if GovUK Frontend Assets already built
+    # if os.path.exists(DIST_PATH):
+    #     print(
+    #         "GovUK Frontend assets already built."
+    #         "If you require a rebuild manually run build.build_govuk_assets"
+    #     )
+    #     return True
+
+    # Download zips from GOVUK_URL
+    # There is a known problem on Mac where one must manually
+    # run the script "Install Certificates.command" found
+    # in the python application folder for this to work.
+
+    print("Downloading static file zip.")
```

Comment:
probably remove print statements, or switch to logging if we need them

Reply:
As this is running locally, print might be more useful